addons/l10n_ch/models/account_bank_statement.py

```diff
@@ -1,25 +0,0 @@
-# -*- coding: utf-8 -*-
-# Part of Odoo. See LICENSE file for full copyright and licensing details.
-
-from odoo import models, fields, api, _
-from odoo.addons.l10n_ch.models.res_bank import _is_l10n_ch_postal
-
-class AccountBankStatementLine(models.Model):
-
-    _inherit = "account.bank.statement.line"
-
-    def _find_or_create_bank_account(self):
-        if self.company_id.account_fiscal_country_id.code in ('CH', 'LI') and _is_l10n_ch_postal(self.account_number):
```

Comment:
This can be removed. It was only useful with postal accounts numbers "01-123-4"

It is a bit weird though, because it is creating a record with `acc_number = postal_number + partner_name but doesn't set the l10n_ch_postal field.

-> :wastebasket: